### ISSUE TITLE
What is recommended amount of INODEs and space for cardano-sl?

### ISSUE THREAD

**Issue by gwpl** [input-output-hk/cardano-sl#2337]
As I've already described here:
https://github.com/input-output-hk/cardano-sl/issues/2305#issuecomment-357654422

I've tried approach of running NixOS appliance on VirtualBox. In mean time I had to grow partition, what I've done by converting from vmdk to vdi, so I could resize.
After growing in size, later it run out in inodex, cardano-sl started throwing "no space on device", and `df -i` gave clear indication that there is no more inodes.

Therefore:

* What is required/recommended amount of inodes and space to make cardano-sl running for a while? (let's say for next year?)

(So, I could add another virtual disk in virtual machine, move all cardano-sl stuff there and make it running).

**Comment by gwpl** [comment]
Update: So far ~1M Inodes seems workable, to I recommend for cardano-sl repo and syncing more to have spare -> at least ~2M Inodes or more. (I personally set >8M Inodes on my VMs)

**Comment by gwpl** [comment]
Here is proposed block store , which could solve the issues : https://github.com/input-output-hk/daedalus/issues/576

**Comment by neongreen** [comment]
Please see: https://github.com/input-output-hk/cardano-sl/pull/2169

tl;dr we are aware of the problem, I've personally written a proposal that was agreed on by other members of the team, now we just need to implement it 😉
